R/palette.R

```diff
@@ -82,8 +82,9 @@ plot.palettehex <- function(x, ...){
   )
   for(i in 1:length(colr)){
     tmp <- colr[[i]]
-    graphics::text(1, i+.1, tolower(tmp$name[[1]]), cex = .7, pos = 2)
-    graphics::text(1, i-.1, tmp$id[[1]], cex = .7, pos = 2)
+    graphics::text(1, i+.1,
+                   paste0(tolower(tmp$name[[1]]), " (", tmp$id[[1]], ")"),
```

Comment:
this is more of a code preference for me, but can you change the `paste0` to `sprintf`? To me it reads clearer what the end string should look like then

`sprintf("%s (%s)", tolower(tmp$name[[1]]), tmp$id[[1]])`

Reply:
Sure! :)